bitratechecker.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python2
+
+from __future__ import print_function
+import sys
+import os
+import glob
+import subprocess
+import json
+import argparse
+
+class Mp3DownloadAnalyzer(object):
+    def __init__(self, mp3_files):
+
+        if not len(mp3_files) > 0:
+            raise ValueError
```

Comment:
Please consider something like that:
```
def __init_(self, mp3_files):
    self.mp3_files = mp3_files

    if not len(self.mp3_files) > 0:
    ...
```

you could then remove ``mp3_files`` from ``__get_media_info()`` and ``__guess_encoder()`` and use ``self.mp3_files`` inside.

Reply:
Good idea. I think `mp3_files` was still used as argument because it was an artifact of an early version, when there were two separate functions without a class.